arm/variants/max3263x_fthr/variant.h

```diff
@@ -79,16 +80,25 @@
 #define PIN_LED     RED_LED
 #define LED_BUILTIN RED_LED
 
+// Push button
+// ----
+#define SW1 P2_3
+#define SW2 NOT_CONNECTED
+#define SW3 NOT_CONNECTED
+// Standardized button names
+#define BUTTON1 SW1
+
+
 // Analog pins
 // -----------
-#define PIN_A0  49
-#define PIN_A1  50
-#define PIN_A2  51
-#define PIN_A3  52
```

Comment:
These pin names must be kept to maintain compatibility with existing programs. Please add them back.

Reply:
readded these definitions.